graderx/graders/import_submissions.py

```diff
@@ -137,7 +139,7 @@ def save_to_lab(self, file_url):
         file_name = drive_service.files().get(
             fileId=file_id).execute()['name']
```

Comment:
I don't really understand what this line executes, is it possible to rename the `execute` function to something more elaborative?

Reply:
.execute() is a part of googleapiclient library

Reply:
it just executes a pre-built request, should I elaborate on what it does somehow in the code or should I assume that the code reader knows the libraries used inside, I'm asking because this exact same case can happen a lot, not just for this case.

Reply:
sorry I didn't notice that this it the Google class. Then it's fine to leave it as is since this is the class doing the details